modules/trusted-firmware-m/Kconfig

```diff
@@ -25,6 +25,7 @@ menuconfig BUILD_WITH_TFM
 	depends on TRUSTED_EXECUTION_NONSECURE
 	depends on TFM_BOARD != ""
 	depends on ARM_TRUSTZONE_M
+	imply BUILD_OUTPUT_HEX
```

Comment:
is `imply` strong enough here ?
If there is not hex output, will the following build / signing / merge stages work as expected ?

Reply:
No, it will fail with "file not found" or something.

Reply:
Is `select` better?

Reply:
in that case, yes.

Reply:
done